src/components/screen/Oauth/modules/PopupModeSwitcher.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
```

Comment:
Instead of empty styles better add root class here. It can define visual context of all nested elements